security/providers/oidc/src/main/java/io/helidon/security/providers/oidc/OidcSupport.java

```diff
@@ -208,6 +219,13 @@ public void update(Routing.Rules rules) {
     }
 
     private void processLogout(ServerRequest req, ServerResponse res) {
+        String tenantName = req.queryParams().first(oidcConfig.paramNameTenant()).orElse(TenantConfigFinder.DEFAULT_TENANT_ID);
```

Comment:
as far as I can see, we can either use a parameter, or a cookie. This is only using parameter.